_posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md

```diff
@@ -0,0 +1,89 @@
+---
+title: New API Blueprint tutorial
```

Comment:
Well the tutorial is really new. There is nothing that stayed the same. Not sure where the issues is – are you saying that a complete rewrite – revamp – of an article isn't a new article (e.g. the use of the word "new" is the problem). 

Would a "New version of API Blueprint tutorial" sound better?